plugins/chat/assets/javascripts/discourse/components/chat/footer/unread-indicator.gjs

```diff
@@ -0,0 +1,55 @@
+import Component from "@glimmer/component";
+import { inject as service } from "@ember/service";
+
+const MAX_UNREAD_COUNT = 99;
+
+export default class ChatFooterUnreadIndicator extends Component {
+  @service chatTrackingStateManager;
+
+  badgeType = this.args.messageType;
```

Comment:
Why do we call it messageType, if it's a badge type? :D

Reply:
I was struggling to find a suitable name for this, I think I changed this a few times already but probably better that they are both the same 👍